tests/framework/ROM/TimeSeries/gold/ARMA/writeSynDataOut_0.csv

```diff
@@ -1,2018 +1,2018 @@
-12.9576208316,1138200.0
-13.1159809732,1138800.0
-13.1549415299,1139400.0
-13.2746393553,1140000.0
-13.4221801887,1140600.0
-13.5705478573,1141200.0
-13.6137671993,1141800.0
-13.5787606896,1142400.0
-13.6764372045,1143000.0
-13.680245129,1143600.0
-13.7022205473,1144200.0
-13.7667603342,1144800.0
-13.698599804,1145400.0
-13.6486748038,1146000.0
-13.4222106006,1146600.0
-13.7247362189,1147200.0
-13.7874016358,1147800.0
-13.6720131106,1148400.0
-13.7089272623,1149000.0
-13.8197029189,1149600.0
-13.8518374456,1150200.0
-13.819094823,1150800.0
-13.9110960411,1151400.0
-13.925471458,1152000.0
-13.9295001934,1152600.0
-13.9188001387,1153200.0
-13.9289129153,1153800.0
-13.9308894156,1154400.0
-13.9385724733,1155000.0
-13.930408376,1155600.0
-13.932700402,1156200.0
-13.9275356065,1156800.0
-13.9298794672,1157400.0
-13.9160325916,1158000.0
-13.9402460218,1158600.0
-13.9339020797,1159200.0
-13.941247937,1159800.0
-13.8954098051,1160400.0
-13.8095672454,1161000.0
-13.8093457963,1161600.0
-13.7656750412,1162200.0
-13.632047303,1162800.0
-13.3835479672,1163400.0
-12.2812477427,1164000.0
-11.5524736911,1164600.0
-11.5038274454,1165200.0
-11.2060657583,1165800.0
-11.3529373931,1166400.0
-11.8172044708,1167000.0
-11.2183325125,1167600.0
-11.6208911149,1168200.0
-11.0779654017,1168800.0
-11.5540067155,1169400.0
-11.4129041258,1170000.0
-10.9827054463,1170600.0
-11.4531507633,1171200.0
-11.8649121677,1171800.0
-11.6189385479,1172400.0
-10.996671739,1173000.0
-11.4952614595,1173600.0
-12.0502136925,1174200.0
-11.4683576045,1174800.0
-11.4850460665,1175400.0
-10.5347606252,1176000.0
-10.2730121352,1176600.0
-11.6977983271,1177200.0
-11.0669176331,1177800.0
-11.0006428599,1178400.0
-11.161581657,1179000.0
-10.4205648155,1179600.0
-10.6847912269,1180200.0
-10.7905138106,1180800.0
-9.93576810621,1181400.0
-9.96952639589,1182000.0
-10.2221874258,1182600.0
-10.1604177213,1183200.0
-10.9878605739,1183800.0
-10.1706189,1184400.0
-10.8874274692,1185000.0
-10.9426568619,1185600.0
-11.1300739209,1186200.0
-10.3778923718,1186800.0
-9.56121245756,1187400.0
-9.68867396669,1188000.0
-8.27757906602,1188600.0
-8.56542753843,1189200.0
-8.59723181206,1189800.0
-8.5511152056,1190400.0
-7.83310998921,1191000.0
-8.52415158828,1191600.0
-9.01103294198,1192200.0
-9.80393267328,1192800.0
-9.63724096412,1193400.0
-9.47600110255,1194000.0
-10.3123450269,1194600.0
-10.6228924316,1195200.0
-10.5814699626,1195800.0
-10.1622373811,1196400.0
-10.6843975939,1197000.0
-10.2124359109,1197600.0
-9.56579421513,1198200.0
-9.76065512395,1198800.0
-10.1575635828,1199400.0
-10.2279600772,1200000.0
-8.72481205432,1200600.0
-9.27662558162,1201200.0
-9.28435523827,1201800.0
-9.40150188181,1202400.0
-9.53478120516,1203000.0
-9.27528070796,1203600.0
-8.54218902649,1204200.0
-8.22587358445,1204800.0
-8.89044178872,1205400.0
-7.82989672105,1206000.0
-7.41015412243,1206600.0
-7.37094560671,1207200.0
-8.31198672291,1207800.0
-8.4074201675,1208400.0
-7.83157028864,1209000.0
-7.50963572981,1209600.0
+Time,Speed
```

Comment:
I know it is not part of this PR and it is not a request change... But, do we need a test of the ARMA that produces 2000 time steps? we should try to shrink this in the future.

Reply:
Yeah, I was thinking this when I saw it too, but I'm insufficiently familiar with all the ARMA parameters to know how they should be changed.  Perhaps this would be good to look at by the project working on the ARMA statistics verification?

Reply:
yep